x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_initializer.tsx

```diff
@@ -108,13 +156,18 @@ export const AnomalySwimlaneInitializer: FC<AnomalySwimlaneInitializerProps> = (
               />
             }
             isInvalid={!isPanelTitleValid}
+            fullWidth
           >
             <EuiFieldText
               id="panelTitle"
               name="panelTitle"
               value={panelTitle}
-              onChange={(e) => setPanelTitle(e.target.value)}
+              onChange={(e) => {
```

Comment:
When editing an existing swim lane, the panel title in the flyout is being reset to the default value. It should display the current title.

<img width="1632" alt="Screenshot 2024-05-07 at 12 55 43" src="https://github.com/elastic/kibana/assets/7405507/2ecad0ca-4770-46e6-a98e-d80f8d35b99d">

Reply:
Fixed in https://github.com/elastic/kibana/pull/182586/commits/5c0714fb07d33eb2df53c025e5d8f09f0c520bbd